src/GraphQl/Resolver/Factory/ItemMutationResolverFactory.php

```diff
@@ -48,10 +49,11 @@
     private $dataPersister;
     private $normalizer;
     private $resourceMetadataFactory;
+    private $mutationLocator;
     private $resourceAccessChecker;
     private $validator;
 
-    public function __construct(IriConverterInterface $iriConverter, DataPersisterInterface $dataPersister, NormalizerInterface $normalizer, ResourceMetadataFactoryInterface $resourceMetadataFactory, ResourceAccessCheckerInterface $resourceAccessChecker = null, ValidatorInterface $validator = null)
+    public function __construct(IriConverterInterface $iriConverter, DataPersisterInterface $dataPersister, NormalizerInterface $normalizer, ResourceMetadataFactoryInterface $resourceMetadataFactory, ContainerInterface $mutationLocator, ResourceAccessCheckerInterface $resourceAccessChecker = null, ValidatorInterface $validator = null)
```

Comment:
This is a BC break. It's allowed for `@experimental` classes but it would be better to avoid it. Can we also made this argument optional? It doesn't have to be mandatory.

Reply:
It can be made optional but do we really want to handle the "breaking" case (when it's null)?